bazel/bazel_unix.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !windows
```

Comment:
This comment doesn't have an equivalent in the windows version. I assume it would need one?

Reply:
I've followed the same build patterns the current project already have and it doesn't seem to have anything on the windows specific files like https://github.com/bazelbuild/bazel-watcher/blob/master/ibazel/ibazel_windows.go.

Ofc We could be adding `// +build windows` for the windows specific implementation. If @achew22 don't think that broke the consistency that he choose to follow on the rest of the project, I can add it